packages/blade/src/components/Card/CardInteractive.stories.tsx

```diff
@@ -0,0 +1,709 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+/* eslint-disable no-alert */
+import type { Meta } from '@storybook/react';
+import React from 'react';
+import type { CardProps } from './Card';
+import {
+  CardBody,
+  Card,
+  CardFooter,
+  CardFooterLeading,
+  CardFooterTrailing,
+  CardHeader,
+  CardHeaderLeading,
+  CardHeaderTrailing,
+  CardHeaderIcon,
+  CardHeaderCounter,
+  CardHeaderBadge,
+} from '.';
+import { Code, Text, Title } from '~components/Typography';
+import { RupeeIcon } from '~components/Icons';
+import { Link } from '~components/Link';
+import { Box } from '~components/Box';
+import { Button } from '~components/Button';
+import { VisuallyHidden } from '~components/VisuallyHidden';
+import { Amount } from '~components/Amount';
+import { isReactNative } from '~utils';
+import StoryPageWrapper from '~utils/storybook/StoryPageWrapper';
+import { Sandbox } from '~utils/storybook/Sandbox';
+
```

Comment:
Please modify all our examples to include Razorpay's contextual Examples. Take anything from payments, X, capital, ezetap, etc.

Reply:
Stories are in context of razorpay only, apart from the Pricing cards. didn't saw any cards being used in the pricing sections.